content/solana-nft.md

```diff
@@ -0,0 +1,618 @@
+
+![Gif of Metaplex Verify Upload](../assets/solana-nft-metaplex-verify-upload.gif)
+
+Now let's mint an NFT from our Candy Machine by running the `mint_one_token` command:
+
+```sh
+ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts mint_one_token \
+    -r, --rpc-url https://metaplex.devnet.rpcpool.com\
+    -k ./private-key.json \
+    -c example
+```
+
+![Gif of Metaplex Mint One](../assets/solana-nft-metaplex-mint-one.gif)
+
+You should now be able to see the newly minted NFT in the Phantom wallet.
+
+![Screenshot of Metaplex NFT](../assets/solana-nft-metaplex-nft.png)
+
+### 7. Candy Machine UI
+
+Now that we know our candy machine works, let's set up a frontend to mint our NFTs using the Candy Machine UI.
+
+Navigate to the candy-machine-ui folder using the following path: `/metaplex/js/packages/candy-machine-ui`.
+
+1. Locate the `.env.example` file
+2. Rename this file to `.env`
+3. Open the `.env` file and update `REACT_APP_CANDY_MACHINE_ID` with the address of the `candyMachine` from the `devnet-example.json` file.
+
+```
+REACT_APP_CANDY_MACHINE_ID=<YOUR CANDY MACHINE PROGRAM ID>
+
+REACT_APP_SOLANA_NETWORK=devnet
+REACT_APP_SOLANA_RPC_HOST=https://metaplex.devnet.rpcpool.com/
+```
+
+From within the `candy-machine-ui` folder, run the following command:
+
+```sh
+yarn install && yarn start
+```
+
+Navigate to [localhost:3000](http://localhost:3000) in the browser and click "MINT" to mint an NFT from your candy machine!
+
+![Gif of Metaplex Mint UI](../assets/solana-nft-metaplex-mint-ui.gif)
+
+### 8. Enable Gatekeeper
+
+Now that we have our Candy Machine UI working, let's update our Candy Machine to enable the gatekeeper setting.
+
+Open the `config.json` file and update the `gatekeeper` field to enable CAPTCHA:
+
+```json
+    "gatekeeper": {
+        "gatekeeperNetwork": "ignREusXmGrscGNUesoU9mxfds9AiYTezUKex2PsZV6",
+        "expireOnUse": true
+    }
+```
+
+Update the Candy Machine by running the `update_candy_machine` command:
+
+```sh
+ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts update_candy_machine \
+    -r, --rpc-url https://metaplex.devnet.rpcpool.com\
+    -k ./private-key.json \
+    -cp config.json \
+    -c example
+```
+
+Note that the path in the command above starts from the root directory of your project. If your command line working directory is different, you'll need to adjust the path accordingly.
+
+Refresh [localhost:3000](http://localhost:3000) in the browser and click the mint button. It should now require a CAPTCHA verification before minting.
+
+![Gif of Metaplex Gatekeeper](../assets/solana-nft-metaplex-gatekeeper.gif)
+
+### 9. Enable Whitelist
+
+Next, let's update our Candy Machine to enable whitelist settings.
+
+We'll start by creating our whitelist token using a helper script included in the starter code.
+
+The script will:
+1. Create a new token mint
+2. Create a new token account
+3. Mint tokens to the token account
+4. Print the whitelist token address to the terminal
```

Comment:
@jamesrp13 replace terminal with console, or command line?

I prefer the term terminal, but this would make our first ever use of it in the lessons so far